.github/workflows/review-code.yml

```diff
@@ -4,152 +4,16 @@ on:
 
-permissions:
-  contents: read
-  pull-requests: write
-
 jobs:
-  validation:
-    name: Validation
-    runs-on: ubuntu-24.04
-    permissions:
-      contents: read
-    outputs:
-      should_review: ${{ steps.validate.outputs.should_review }}
-
-    steps:
-      - name: Check PR requirements
-        id: check-pr
-        run: |
-          if [ "${{ github.event.pull_request.draft }}" == "true" ]; then
-            echo "⚠️  Validation: PR is a draft - skipping review"
-            echo "pr_valid=false" >> $GITHUB_OUTPUT
-          else
-            echo "✅ Validation: PR is ready for review"
-            echo "pr_valid=true" >> $GITHUB_OUTPUT
-          fi
-
-      - name: Check for Azure credentials
-        id: check-azure-secret
-        env:
-          _AZURE_SUBSCRIPTION_ID: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
-          _AZURE_TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
-          _AZURE_CLIENT_ID: ${{ secrets.AZURE_CLIENT_ID }}
-        run: |
-          if [ -n "$_AZURE_SUBSCRIPTION_ID" ] && [ -n "$_AZURE_TENANT_ID" ] && [ -n "$_AZURE_CLIENT_ID" ]; then
-            echo "credentials_valid=true" >> $GITHUB_OUTPUT
-            echo "✅ Validation: Azure credentials available"
-          else
-            echo "credentials_valid=false" >> $GITHUB_OUTPUT
-            echo "⚠️  Validation: Azure credentials not available"
-            echo "This is expected for external contributors or forks"
-          fi
-
-      - name: Check if prompt file exists
-        id: check-prompt
-        env:
-          GH_TOKEN: ${{ github.token }}
-        run: |
-          # Use GitHub API to check file existence WITHOUT checkout
-          FILE_PATH=".claude/prompts/review-code.md"
-          REPO="${{ github.repository }}"
-          REF="${{ github.event.pull_request.head.sha }}"
-
-          # Check if file exists via API
-          if gh api "repos/$REPO/contents/$FILE_PATH?ref=$REF" --silent 2>/dev/null; then
-            echo "prompt_exists=true" >> $GITHUB_OUTPUT
-            echo "✅ Found $FILE_PATH in $REPO"
-          else
-            echo "prompt_exists=false" >> $GITHUB_OUTPUT
-            echo "⚠️  Validation: No $FILE_PATH found - skipping Claude review"
-          fi
-
-      - name: Set validation result
-        id: validate
-        run: |
-          if [ "${{ steps.check-pr.outputs.pr_valid }}" == "true" ] && \
-             [ "${{ steps.check-azure-secret.outputs.credentials_valid }}" == "true" ] && \
-             [ "${{ steps.check-prompt.outputs.prompt_exists }}" == "true" ]; then
-            echo "should_review=true" >> $GITHUB_OUTPUT
-            echo "✅ Validation passed - code review will proceed"
-          else
-            echo "should_review=false" >> $GITHUB_OUTPUT
-            echo "⚠️  Validation failed - code review will be skipped"
-          fi
-
   review:
-    name: Review
-    runs-on: ubuntu-24.04
-    needs: validation
-    if: needs.validation.outputs.should_review == 'true'
+    name: Review Code
```

Comment:
```suggestion
    name: Review
```

Reply:
⛏️ But I like to keep job names tighter. This will show as "Review Code > Review" in the UI. At any rate it's not fully capitalized unless it's a workflow name.